composer.json

```diff
@@ -14,7 +14,9 @@
         "aura/di": "dev-develop-2",
         "bodom78/kohana-imagefly": "dev-master",
         "league/oauth2-server": "~3.2.4",
-        "league/flysystem": "0.4.*",
+        "league/flysystem-aws-s3-v3": "1.0.4",
```

Comment:
PHP 5.4 security support ends in 9 days. We could so dropping it might be reasonable: http://php.net/supported-versions.php